### PR TITLE
Influxdb: Fix cascader when doing log query in explore

### LINKED ISSUE #21744: Influxdb: measurement picker does not work
**What happened**:
When trying to search for logs with elastic the measurement picker does not fill in field into the request:

Screenshot 2020-01-25 at 14 29 27

This creates request like this `SELECT "undefined" FROM "logs" WHERE time >= now() - 1h LIMIT 1000` and no logs in the response.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

This seems to be already existing issue in 6.5.3, still persists in 6.6 beta. In master there are some underlying issues making it a bit more broken, but when fixed with https://github.com/grafana/grafana/pull/21743 we will get to this same issue still.

**Environment**:
- Grafana version:
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by aocenas**
Closes: https://github.com/grafana/grafana/issues/21744

Subitems were passed in wrong attribute of the options object. Added tests and simple influx datasource mock.

### CODE REVIEW

File: public/app/plugins/datasource/influxdb/datasource.mock.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      return this.getMeasurementFiels(query);
+    }
+  }
+
+  private getMeasurements(): FieldReturnValue[] {
+    return Object.keys(this.measurements).map(key => ({ text: key }));
+  }
+
+  private getMeasurementFiels(query: string): FieldReturnValue[] {
+    const match = query.match(/SHOW FIELD KEYS FROM \"(.+)\"/);
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/influxdb/datasource.mock.ts]: Should this be `getMeasurementFields`?

Comment (aocenas) [reply public/app/plugins/datasource/influxdb/datasource.mock.ts]: Not sure what you mean. Do you have some other name in mind for the function or do you mean the regex does not seem to match the name of the method?

File: public/app/plugins/datasource/influxdb/datasource.mock.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    } else {
+      return this.getMeasurementFiels(query);
+    }
+  }
+
+  private getMeasurements(): FieldReturnValue[] {
+    return Object.keys(this.measurements).map(key => ({ text: key }));
+  }
+
+  private getMeasurementFiels(query: string): FieldReturnValue[] {
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/influxdb/datasource.mock.ts]:
```suggestion
  private getMeasurementFields(query: string): FieldReturnValue[] {
```

File: public/app/plugins/datasource/influxdb/datasource.mock.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ * Datasource mock for influx. At the moment this only works for queries that should return measurements or their
+ * fields and no other functionality is implemented.
+ */
+export class InfluxDatasourceMock {
+  constructor(private measurements: Measurements) {}
+  metricFindQuery(query: string) {
+    if (isMeasurementsQuery(query)) {
+      return this.getMeasurements();
+    } else {
+      return this.getMeasurementFiels(query);
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/influxdb/datasource.mock.ts]:
```suggestion
      return this. getMeasurementFields(query);
```

**Review by kaydelaney — APPROVED** [review]

**Pull request merged**